python/hail/context.py

```diff
@@ -178,15 +178,27 @@ def default_reference():
 def get_reference(name):
     """Return the reference genome corresponding to `name`.
 
+    Hail built-in references are ``'GRCh37'`` and ``GRCh38'``. The contig names
+    and lengths were obtained from the GATK resource bundle for
+    `human_g1k_v37.dict <ftp://gsapubftp-anonymous@ftp.broadinstitute.org/bundle/b37/human_g1k_v37.dict>`__
+    and `Homo_sapiens_assembly38.dict <ftp://gsapubftp-anonymous@ftp.broadinstitute.org/bundle/hg38/Homo_sapiens_assembly38.dict>`__.
+
     If `name` is ``'default'``, return the reference from :func:`.default_reference`.
 
+    Parameters
+    ----------
+    name : :obj:`str`
+        Reference genome name. This can be a previously loaded reference genome
```

Comment:
I'd change to:
Name of a previously loaded reference genome or one of Hail's built-in references: ``'GRCh37'``, ``'GRCh38'``, and ``'default'``.